app/models/card.rb

```diff
@@ -2,16 +2,21 @@ class Card < ActiveRecord::Base
 validates :original_text, :translated_text, presence: true, uniqueness: { case_sensitive: false }
 validates :review_date, presence: true
 validate :must_not_be_equal
+scope :expired, -> { where('review_date <= ?', DateTime.now) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.